examples/issue-bbs-credentials/index.ts

```diff
@@ -0,0 +1,91 @@
+import {
+	CREDENTIALS_CONTEXT_V1_URL,
+	KeyType,
+	SECURITY_CONTEXT_BBS_URL,
+	TypedArrayEncoder,
+} from "@aries-framework/core";
```

Comment:
Should be credo-ts right?

```suggestion
} from "@credo-ts/core";
```